core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -120,7 +120,7 @@
 
 #define CFG_TEE_CORE_NB_CORE	8
 
-#define CFG_TEE_RAM_VA_SIZE	(1024 * 1024)
+#define CFG_TEE_RAM_VA_SIZE	(2 * 1024 * 1024)
```

Comment:
Minor: the memory map above (comment block) needs fixing.

Reply:
Actually, i don't think this change is required. 1MB is fine for current. Increase to 2MB should not hurt (but the layout description pointed by @jforissier) and allows to test the change.

Reply:
`s/0x3F10_0000/0x3F20_0000/`

I think we all agree 2M is a better default value if only for exercising the new code.
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`